.github/ISSUE_TEMPLATE/release.yaml

```
@@ -0,0 +1,78 @@
+name:  Release
+description: Release/Pre-release of a version.
+title: "Release Version(e.g: 2.4.0)"
+labels: [release]
+body:
+- type: dropdown
+  id: release_type
+  attributes:
+    label: Release Type
+    description: which type of release is this release?
+    options:
+    - major release
+    - minor release
+    - patch release
+  validations:
+    required: true
+- type: input
+  id: release_version
+  attributes:
+    label: Relase Version
+    placeholder: "e.g: 2.4.0"
+  validations:
+    required: true
+- type: checkboxes
+  id: release_tests
+  attributes:
+    label: "**For all releases** Github Workflow Test Matrix Checkup"
+    options:
+      - label: check the testing workflow ([.github/workflows/test.yaml](/Kong/kubernetes-ingress-controller/blob/main/.github/workflows/test.yaml)) and ensure that all matrix versions ([.github/workflows/e2e.yaml](/Kong/kubernetes-ingress-controller/blob/main/.github/workflows/e2e.yaml) and [.github/workflows/release.yaml](/Kong/kubernetes-ingress-controller/blob/main/.github/workflows/release.yaml))  are up to date for various component releases. If there have been any new releases (major, minor or patch) of those components since the latest version seen in that configuration make sure the new versions get added before proceeding with the release. Remove any versions that are no longer supported by the environment provider.
+      - label: Kubernetes
+      - label: Istio
+- type: checkboxes
+  id: release_branch
+  attributes:
+    label: "**For all releases** Create Release Branch"
+    options: 
+      - label: "ensure that you have up to date copy of `main`: git fetch --all"
```

Partly incorporating the old comment and adjusting it a bit so that the commands definitely start your branch from main and local main has upstream changes:

```suggestion
      - label: "ensure that you have up to date copy of `main`: git checkout main; git pull"
```

---

done